src/components/styleguide/NavBar.jsx

```diff
@@ -1,7 +1,24 @@
 import React from 'react';
 import Radium from 'radium';
```

Comment:
We can probably delete `Radium` here as it does not appear that we are using it.